### DISCUSSION TITLE
OpenSSL 3.0.11 function for PEM_read_RSAPrivateKey

### DISCUSSION THREAD

**Discussion by debananda-pal** [openssl/openssl#25034] [Q&A] [answered]
Hello Sir/Madam,

We have migrated code from Buster to Bookworm and using OpenSSL 3.0.11.
Earlier we used OpenSSL 1.1xx version and used PEM_read_RSAPrivateKey().
As this function is deprecated, kindly let me know the alternate function in OpenSSL 3.0.11

Regards,
D Pal

**Comment by t8m** [comment, marked as answer]
The replacement function is PEM_read_PrivateKey().

**Reply by debananda-pal** [reply]
Thank you very much.
Please let me know the alternate function for PEM_write_RSAPrivateKey().
Is the alternate function PEM_write_PrivateKey(), please let me know.

**Reply by t8m** [reply]
Yes, exactly.

**Reply by debananda-pal** [reply]
Thank you.
